nbs/core.ipynb

```diff
@@ -292,7 +292,6 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #599.                    futr_df: Optional[DataFrame] = test
</pre><p>Remove Optinal[DataFrame]?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>This is in case we adopt mypy, if we ran it it would complain about futr_df being declared as DataFrame and then assigned as None in the else below.</p>